<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PictureEditor</class>
 <widget class="QWidget" name="PictureEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PictureEditor</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <widget class="QLabel" name="label_pic">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>704</width>
          <height>512</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_hs">
       <property name="text">
        <string>直方图</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_bp">
       <property name="text">
        <string>反向投影</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_load">
       <property name="text">
        <string>加载图片</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_original">
       <property name="text">
        <string>原图显示</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_erosion">
       <property name="text">
        <string>图像腐蚀</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_blur">
       <property name="text">
        <string>图像模糊</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_edge">
       <property name="text">
        <string>边缘检测</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_chanR">
       <property name="text">
        <string>R通道</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_weightMix">
       <property name="text">
        <string>加权混合</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_dft">
       <property name="text">
        <string>离散傅里叶</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_box">
       <property name="text">
        <string>方框滤波</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_mean">
       <property name="text">
        <string>均值滤波</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_gauss">
       <property name="text">
        <string>高斯滤波</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_sobel">
       <property name="text">
        <string>sobel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_laplacian">
       <property name="text">
        <string>Laplacian</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_scharr">
       <property name="text">
        <string>Scharr</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_houghlines">
       <property name="text">
        <string>HoughLines</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_remap">
       <property name="text">
        <string>Remap</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_warp">
       <property name="text">
        <string>仿射变换</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_hist">
       <property name="text">
        <string>EqualizeHist</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_contour">
       <property name="text">
        <string>轮廓</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
